specs/proto/wire.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+// Define wire messages for transaction types.
+// https://github.com/lazyledger/lazyledger-specs/blob/master/specs/data_structures.md#signedtransactiondata
+
+message TransactionFee {
+  // Max base fee rate (burnt)
+  uint64 base_rate_max = 1;
+  // Max tip fee rate (rewarded to block proposer)
+  uint64 tip_rate_max = 2;
+}
+
+message Decimal {
+  uint64 todo = 1;
```

Comment:
Can you link to issue #120 in a `// comment` here too?

Reply:
Linked in 787aa4c.